Week2/homework/maartjes-work.js

```diff
@@ -45,17 +45,20 @@ const tuesday = [
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
-function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+function computeEarnings(tasks) {
+  return tasks
+    .map(hour => hour.duration / 60)
```

Comment:
When you are mapping an array of `tasks`, the callback function that you pass to the `.map()` method get a single `task` at a time. Therefore this parameter should be called `task` instead of `hour`:

```js
.map(task => task.duration / 60)
```

Reply:
Thanks Jim for your review ^_^